go.mod

```diff
@@ -30,3 +30,5 @@ require (
 go 1.15
 
 replace github.com/lightninglabs/loop/swapserverrpc => ./swapserverrpc
+
+replace github.com/lightningnetwork/lnd => github.com/carlakc/lnd v0.4.2-beta.0.20220127131004-c15d010ba203
```

Comment:
can be pointed at master now!

Reply:
I pointed it to your commit and we can update once 0.14.2-beta is tagged.

Reply:
Can point to the rc now that it's tagged!

Reply:
Unfortunately it's not in the tagged rc1.

Reply:
Ah yes of course 🤦‍♀️  I think we should point to lnd master (or the commit specifically `v0.14.1-beta.0.20220128084429-065412b9487b`) over my branch? 

Assuming we'll do a loop release before this is included in a tagged version of lnd? I'm sure folks would prefer the dep is lnd than my fork.

Reply:
Updated to LND master.